capstone-frontend/src/components/CreateList.jsx

```diff
@@ -64,22 +135,80 @@ class CreateList extends Component {
+                    <Spinner
+                      as="span"
+                      animation="border"
+                      size="lg"
+                      role="status"
+                      aria-hidden="true"
+                      className="my-5"
+                    />
+                  </div>}
+              </Form.Group>
+
+              {
+                this.state.displayBooks &&
+
+                <div>
+                  <h3 clasName="my-4 px-4">Search Results</h3>
+                  <Row className="px-3 text-center">
+                    {this.state.searchResults.map(book =>
+                      <Col md={3} className="px-2 my-0 border" style={{ borderColor: "#ccc" }} key={book.id}>
+                        <img className="img-responsive mt-3 p-0 rounded" src={book.thumbnailLink} alt={book.title} />
+                        <h5 className="mt-4"> {book.title} </h5>
+                        <p className="my-1"> {book.authors.join(', ')} </p>
+                        {this.state.addedBooksIDs.includes(book.id) ?
+                          <Button className="my-5" variant="danger" onClick={() => this.removeBookFromList(book)}>Remove Book</Button>
+                          :
+                          <Button className="my-5" onClick={() => this.addBookToList(book)}>Add to Booklist</Button>}
+                      </Col>
+                    )}
+                  </Row>
+                </div>
+              }
+
+              {
+                (this.state.addedBooks.length !== 0) &&
+
+                <div>
+                  <h2 className="text-center my-4 px-4 ">Added Books</h2>
+                  <Row className="text-center px-3">
+                    {this.state.addedBooks.map(addedBook =>
+
+                      <Col md={3} className="px-2 my-0 border" style={{ borderColor: "#ccc" }} key={addedBook.id}>
+                        <img className="img-responsive mt-3 p-0 rounded" src={addedBook.thumbnailLink} alt={addedBook.title} />
+                        <h5 className="mt-4"> {addedBook.title} </h5>
+                        <p className="my-1"> {addedBook.authors.join(', ')} </p>
+                        <Button className="my-5" variant="danger" onClick={() => this.removeBookFromList(addedBook)}>Remove Book</Button>
+                      </Col>
+                    )}
+                  </Row>
+                </div>
+              }
+
+              <div className="text-center">
+                <Button className="text-center" variant="primary" type="submit" onClick={() => this.handleSubmit()} disabled={this.state.creatingBookList}>
+                  Create Booklist
+                    {this.state.creatingBookList &&
+                    <Spinner
+                      as="span"
+                      animation="border"
+                      size="sm"
+                      role="status"
+                      aria-hidden="true"
+                      className="ml-4"
+                    />}
```

Comment:
If we are going to use these Spinners with these constant similar values across the app, we should consider making a component Wrapper that will define this for us so we can just use something like `<CustomSpinner/> instead of having to do this every time